### ISSUE TITLE
[Account Recovery/Close] Close/Recovery Fails + missing funds?

### ISSUE THREAD

**Issue by levmi** [lightninglabs/pool#400]
_Apologies in advance if I'm doing something dumb_ 😅 But, if I got into this state, I'm sure other users can as well. 

A month or so ago, I deposited ~7.8m sats to my Pool account (relevant transaction: https://mempool.space/tx/59132bc3261f7fbb67d8da328593a80907bce4d485b88e293c488d0afdf93c38). The account balance never updated to reflect the deposited funds. And then, a new account was seemingly created which wiped my previous balance and reduced it significantly. It was a weird flow, but I have been too busy to look into it in detail until today.

Today, I went through the flows of looking at my accounts via the CLI. I see two accounts present, which makes sense given the above description. The trader keys for those accounts are `03958b530db3cc208cb47881bef757c37ed0aaa366502793463196c9428d596266` with a balance of `39861` and `03d6cd0a388e45851ce094b6d75833b2428a81849af96c5aeb6b751d3ec901fa9c` with a balance of `18167832`. I'm pretty sure that neither of those amounts account for the deposited 7.8m sats. I closed the first account with no problem. 

I attempted to recover the second account and got back the following error: `[pool] rpc error: code = Unknown desc = error performing recovery: could not recover account 03d6cd0a388e45851ce094b6d75833b2428a81849af96c5aeb6b751d3ec901fa9c: account 03d6cd0a388e45851ce094b6d75833b2428a81849af96c5aeb6b751d3ec901fa9c is already subscribed, cannot recover`

Then, I attempted to close the second account and got back the following error: `rpc error: code = Unknown desc = transaction rejected: output already spent`

So, I'm in a bit of a stuck state with that second account. Additionally, even if I could recover the account, I'm not sure where those 7.8m sats went 😭

**Comment by positiveblue** [comment]
Hello @levmi, no worries you should be able to recover your funds without any problem.

You can have multiple accounts per pool daemon. Those accounts are linked to the LND node that you are using as a backend, did you run multiple pool clients using the same node?

Before performing a recovery, you need to delete the file with your client data (`pool.db`). You can then start the recovery process but it will cancel all your open orders and you will also lose the data about previous matches.

After the recovery process you should have the latest state for each one of your accounts.

P.S: I do not see any output of ~7.8M in the tx that you attached, did you copy and paste the right tx?

**Comment by levmi** [comment]
Okay, sweet. Jordi and I did some chat based debugging. There was a weird issue whereby the CLI wasn't returning the proper account details and the latest tx was invalid. He believes that he found why that might be so there should be an update on that front in the form of a subasta issue. But, I went through the discussed path of deleting the db, account recovery, etc. and seems like it all went fine, just waiting for confs. So, I'll close this issue. Thanks!
